python/py_helpers.py

```diff
@@ -99,6 +131,14 @@ def get_variable(self, name):
 
     def has_function(self, name):
         return self.find_function(name) != Node()
+    
+    def has_decorators(self, *args):
+        if not isinstance(self.tree, ast.FunctionDef):
+            return False
+        if len(args) == len(self.tree.decorator_list) \
+        and all(node.id in args for node in self.tree.decorator_list):
+            return True
+        return False
```

Comment:
```suggestion
        if len(args) != len(self.tree.decorator_list):
            return False
        return all(node.id in args for node in self.tree.decorator_list)
```
This reads better, I think.